perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/UserAttributeProcessor.java

```diff
@@ -18,21 +21,22 @@
 
 	@Autowired
 	protected PerunUser perunUser;
-	
+
 	private static Pattern userSetPattern = Pattern.compile(" set for User:\\[(.*)\\]");
 	private static Pattern userRemovePattern = Pattern.compile(" removed for User:\\[(.*)\\]");
 	private static Pattern userAllAttrsRemovedPattern = Pattern.compile("All attributes removed for User:\\[(.*)\\]");
+	private static Pattern userVirtualChangePattern = Pattern.compile(" changed for User:\\[.*\\]");
```

Comment:
Please change ``.*`` -> ``(.*)``. Its not necessary, but I like all regexes looking the same at first sight, so that you can focus on rest of regex content, if its not working properly.